WordPress/src/androidTest/java/org/wordpress/android/e2e/ContactUsTests.java

```diff
@@ -0,0 +1,84 @@
+package org.wordpress.android.e2e;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.wordpress.android.R;
+import org.wordpress.android.e2e.flows.LoginFlow;
+import org.wordpress.android.e2e.pages.ContactSupportScreen;
+import org.wordpress.android.support.BaseTest;
+
+import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_EMAIL;
+import static org.wordpress.android.support.WPSupportUtils.pressBackUntilElementIsDisplayed;
+
+public class ContactUsTests extends BaseTest {
+    @Before
+    public void setUp() {
+        logoutIfNecessary();
+    }
+
+    @After
+    public void tearDown() {
+        pressBackUntilElementIsDisplayed(R.id.continue_with_wpcom_button);
+    }
+
+    @Test
+    public void sendButtonEnabledWhenTextIsEntered() {
+        try {
+            new LoginFlow()
+                .chooseContinueWithWpCom()
+                .tapHelp()
+                .assertHelpAndSupportScreenLoaded()
+                .openContactUs()
+                .assertContactSupportScreenLoaded()
+                .assertSendButtonDisabled()
+                .setMessageText("Hello")
+                .assertSendButtonEnabled()
+                .setMessageText("")
+                .assertSendButtonDisabled();
+        } finally {
+            new ContactSupportScreen().goBackAndDeleteUnsentMessageIfNeeded();
+        }
+    }
+
+    @Ignore("As long as CI does not use gradle.properties from MobileSecrets")
+    @Test
+    public void messageCanBeSent() {
+        String userMessageText = "Please ignore, this is an automated test.";
+        String automatedReplyText = "Mobile support will respond as soon as possible, "
+                                    + "generally within 48-96 hours. "
+                                    + "Please reply with your site address (URL) "
+                                    + "and any additional details we should know.";
```

Comment:
I have this test failing since the message have slightly changed due to holidays. We might want to do a partial check as the two last lines remained the same.

![image](https://user-images.githubusercontent.com/42008628/147502692-0a17527c-7352-43ac-a2a9-40d50b9dbd79.png)

Reply:
Yes. So far, I'm not sure if it makes sense to maintain this test (or even to keep it) - I probably firstly need to ask if Zendesk connection is ever planned on CI.

Reply:
We have it connected for WPiOS because it was not possible to even check if the screen had been loaded. From what I can recall it is possible but not desirable. Given it is an identified critical flow I'd ask @jkmassel to weigh in. =)

Reply:
We could also use a less sophisticated check and ensure we get _any_ text in `R.id.request_system_message_text`. 🤷‍♂️